src/Compilers/Core/Portable/InternalUtilities/Debug.cs

```diff
@@ -5,19 +5,31 @@
 using System;
 using System.Diagnostics;
 using System.Diagnostics.CodeAnalysis;
+using System.Runtime.CompilerServices;
+using System.Text;
 
 namespace Roslyn.Utilities
 {
     internal static class RoslynDebug
     {
         /// <inheritdoc cref="Debug.Assert(bool)"/>
         [Conditional("DEBUG")]
-        public static void Assert([DoesNotReturnIf(false)] bool b) => Debug.Assert(b);
+        public static void Assert([DoesNotReturnIf(false)] bool condition) => Debug.Assert(condition);
 
         /// <inheritdoc cref="Debug.Assert(bool, string)"/>
         [Conditional("DEBUG")]
-        public static void Assert([DoesNotReturnIf(false)] bool b, string message)
-            => Debug.Assert(b, message);
+        public static void Assert([DoesNotReturnIf(false)] bool condition, string message)
+            => Debug.Assert(condition, message);
+
+        /// <inheritdoc cref="Debug.Assert(bool, string)"/>
+        [Conditional("DEBUG")]
+        public static void Assert([DoesNotReturnIf(false)] bool condition, [InterpolatedStringHandlerArgument(nameof(condition))] ref AssertInterpolatedStringHandler message)
```

Comment:
>Assert [](http://example.com/codeflow?start=27&length=6)

Should we polyfill this as a method in `System.Diagnostics.Debug` rather than `Roslyn.Utilities.RoslynDebug`, to avoid changing call sites and avoid overlooking call sites?

Reply:
It's a `static` method so can't polyfill until we get extension everything 😦